### PR TITLE
Fix various typos in the documentation

### PR DESCRIPTION
**Pull request by blabber**
`aspell -d en check` all the rst files.

### CODE REVIEW

File: docs/Playing/faq.rst [hunk 1]
```diff
@@ -783,7 +783,7 @@ to get help on a particular setting, or
 to change a setting to a particular value. After the game begins you may still change some settings, but not
 others.
 
-You can create rulesets or :strong:`modpacks` - alternative sets of units, buildings, and technologies.
+You can create rulesets or :strong:`mudpacks` - alternative sets of units, buildings, and technologies.
```
Comment (lmoureaux) [inline docs/Playing/faq.rst:786]: No :D

**Review by lmoureaux — CHANGES_REQUESTED** [review]
Just one change of `modpack` to `mudpack` that I don't really like, the rest looks good :+1:

**Comment by blabber** [comment]
Fixed the mudpacks by force-push.

File: docs/Playing/faq.rst (hunk 1 above)
Comment (blabber) [reply docs/Playing/faq.rst]: :'D

**Comment by psampathkumar** [comment]
While you are at it.. Add yourself to the author file 😄

**Review by lmoureaux — APPROVED** [review]

**Pull request merged**